subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/TransformationNode.java

```diff
@@ -312,14 +332,28 @@ public TransformationSubject getTransformedSubject() {
 
     private static class OperationDetails implements ExecuteScheduledTransformationStepBuildOperationType.Details {
 
+        private final String buildPath;
+        private final long transformationId;
         private final String transformerName;
         private final String subjectName;
 
-        public OperationDetails(String transformerName, String subjectName) {
+        public OperationDetails(String buildPath, long transformationId, String transformerName, String subjectName) {
+            this.buildPath = buildPath;
+            this.transformationId = transformationId;
             this.transformerName = transformerName;
             this.subjectName = subjectName;
         }
 
+        @Override
+        public String getBuildPath() {
+            return buildPath;
+        }
+
+        @Override
+        public long getTransformationId() {
+            return transformationId;
+        }
```

Comment:
To recap, instead of these two we should have a `getTransformationIdentity()` (or `getIdentity()`, but then let's rename `Task.getTaskIdentity()`, too).